realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -489,4 +502,26 @@ public RealmConfiguration getConfiguration() {
     public ColumnIndices[] getTypedColumnIndicesArray() {
         return typedColumnIndicesArray;
     }
+
+    /**
+     * @return the total global ref count.
+     */
+    private int getTotalGlobalRefCount() {
+        int totalRefCount = 0;
+        for (RefAndCount refAndCount : refAndCountMap.values()) {
+            totalRefCount += refAndCount.globalCount;
+        }
+
+        return totalRefCount;
+    }
+
+    /**
+     * If a Realm instance is GCed but `Realm.close()` is not called before, we still want to track the cache for
```

Comment:
It is available. First there will be a log to say Realm instance is not closed.
then user can call `Realm.getLocalInstanceCount()` and `Realm.getGlobalInstanceCount()` for how many references are not released just like before.